cmd/cstor-pool-mgmt/pool/v1alpha2/pool_utils.go

```diff
@@ -0,0 +1,127 @@
+package v1alpha2
+
+import (
+	"strings"
+
+	api "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha2"
+	zpool "github.com/openebs/maya/pkg/apis/openebs.io/zpool/v1alpha1"
+	blockdevice "github.com/openebs/maya/pkg/blockdevice/v1alpha2"
+	env "github.com/openebs/maya/pkg/env/v1alpha1"
+	zfs "github.com/openebs/maya/pkg/zfs/cmd/v1alpha1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func getPathForBdevList(bdevs []api.CStorPoolClusterBlockDevice) ([]string, error) {
+	var vdev []string
+	var err error
+
+	for _, b := range bdevs {
+		path, er := getPathForBDev(b.BlockDeviceName)
+		if er != nil {
+			err = ErrorWrapf(err, "Failed to fetch path for bdev {%s} {%s}", b.BlockDeviceName, er.Error())
+			continue
+		}
+		vdev = append(vdev, path)
+	}
+	return vdev, err
+}
+
+func getPathForBDev(bdev string) (string, error) {
+	// TODO
+	// replace `NAMESPACE` with env variable from CSP deployment
+	bd, err := blockdevice.NewKubeClient().
+		WithNamespace(env.Get("NAMESPACE")).
+		Get(bdev, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	return bd.Spec.Path, nil
+}
+
+func checkIfPoolPresent(name string) bool {
+	if _, err := zfs.NewPoolGProperty().
```

Comment:
lets make NewPoolGProperty as NewPoolGetProperty.. similarly for SetProperty also